lib/player.rb

```diff
@@ -168,14 +208,38 @@ def find_cells_hit
     end
   end
 
+  def generate_adjactent_letters(coord)
+    ord_letter_array = coord_letter(coord).split('').map { |letter| letter.ord }
+    same_double = ord_letter_array.map { |ord| ord.chr }.join
```

Comment:
We could use coord here

Reply:
SOLID idea!  but alas, it returns an extra number, broke a test. Tried it and reverted.